runtime/tests/port/si.c

```diff
@@ -2280,6 +2280,27 @@ j9sysinfo_test_get_l1dcache_line_size(struct J9PortLibrary *portLibrary)
 	return reportTestExit(portLibrary, testName);
 }
 
+#if defined(J9VM_OPT_CRIU_SUPPORT)
+I_32
+j9sysinfo_test_get_process_start_time(struct J9PortLibrary *portLibrary)
+{
+	PORT_ACCESS_FROM_PORT(portLibrary);
+	const char *testName = "j9sysinfo_test_get_process_start_time";
+	UDATA pid = j9sysinfo_get_pid();
+	U_64 processStartTimeInNanoseconds = 0;
+	I_32 rc = j9sysinfo_get_process_start_time(pid, &processStartTimeInNanoseconds);
+	if ((0 != rc) || (0 == processStartTimeInNanoseconds)) {
+		outputErrorMessage(
+			PORTTEST_ERROR_ARGS,
+			"j9sysinfo_get_process_start_time, pid=%d, processStartTimeInNanoseconds=%llu, rc=%d.\n",
```

Comment:
The template doesn't match the arguments; OMR formats should be used (i.e. `%zu` for `UDATA`).